### PR TITLE
Restrict Gallery to phone-sized screens

### PR DESCRIPTION
**Pull request by jason-simmons**
See https://github.com/flutter/flutter/issues/5570

### CODE REVIEW

**Comment by jason-simmons** [comment]
@sethladd

File: examples/flutter_gallery/android/AndroidManifest.xml [hunk 1]
```diff
@@ -8,6 +8,19 @@
     <uses-sdk android:minSdkVersion="14" android:targetSdkVersion="21" />
     <uses-permission android:name="android.permission.INTERNET"/>
 
+    <compatible-screens>
```
Comment (sethladd) [inline examples/flutter_gallery/android/AndroidManifest.xml]: Maybe add a comment for what we're doing here? Perhaps "White-listing phones, until such time as Flutter's widgets and layouts have been tested for tablets."

**Comment by sethladd** [comment]
lgtm with small note

File: examples/flutter_gallery/android/AndroidManifest.xml (hunk 1 above)
Comment (jason-simmons) [inline examples/flutter_gallery/android/AndroidManifest.xml]: done

**Comment by sethladd** [comment]
lgtm

**Pull request merged**